src/HaskellCI/GitHub/Yaml.hs

```diff
@@ -30,11 +30,17 @@ data GitHubJob = GitHubJob
     { ghjName      :: String
     , ghjRunsOn    :: String
     , ghjContainer :: Maybe String
-    , ghjMatrix    :: [M.Map String String]
+    , ghjMatrix    :: [M.Map String GitHubMatrixValue]
```

Comment:
No, I think the whole `M.Map String` is a bit to dynamic to begin with. (I was lazy, sorry).

```haskell
ghcMatirx :: [GitHubMatrixEntry]

data GitHubMatrixEntry = GitHubMatrixEntry
    { ghmeGhcVersion :: String -- or Version?
    , ghcmeContinueOnError :: Bool
    }
```

Reply:
Alright, I've incorporated your suggestion.